WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignsCardView.swift

```diff
@@ -69,6 +76,10 @@ final class DashboardBlazeCampaignsCardView: UIView {
         frameView.onHeaderTap = { [weak self] in
             self?.showCampaignList()
         }
+
+        campaignView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(campainViewTapped)))
+        campaignView.isAccessibilityElement = true
+        campaignView.accessibilityTraits = .allowsDirectInteraction
```

Comment:
Q: These properties are already being set within `CampaignView` - is there a reason for explicitly setting them again here?

Reply:
Good catch, thanks. I forgot to remove it from the previous commit.